gallery/v2_transforms/plot_transforms_v2.py

```diff
@@ -7,103 +7,253 @@
     Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms_v2.ipynb>`_
     or :ref:`go to the end <sphx_glr_download_auto_examples_v2_transforms_plot_transforms_v2.py>` to download the full example code.
 
-Most computer vision tasks are not supported out of the box by ``torchvision.transforms`` v1, since it only supports
-images. ``torchvision.transforms.v2`` enables jointly transforming images, videos, bounding boxes, and masks. This
-example showcases the core functionality of the new ``torchvision.transforms.v2`` API.
+This example illustrates all of what you need to know to get started with the
+new :mod:`torchvision.transforms.v2` transforms. We'll cover simple tasks like
```

Comment:
I'm not pretending for a better wording, but "transforms.v2 transforms" does not read very well:
```suggestion
new :mod:`torchvision.transforms.v2` API. We'll cover simple tasks like
```